v-dev.sql

```diff
@@ -72,4 +72,15 @@ UPDATE openchpl.testing_lab
 SET deleted = false;
 ----------------------------------------------------
 -- END OCD-1665 Retire ACBs and ATLS
+----------------------------------------------------
+
+----------------------------------------------------
+-- OCD-2554: Remove old notification infrastructure
+----------------------------------------------------
+DROP TABLE IF EXISTS openchpl.notification_type_recipient_map;
+DROP TABLE IF EXISTS openchpl.notification_recipient;
+DROP TABLE IF EXISTS openchpl.notification_type_permission;
+DROP TABLE IF EXISTS openchpl.notification_type;
```

Comment:
Does it need a cascade or anything to get rid of the triggers?

Reply:
Nope! They go when the table gets dropped.